.changeset/spicy-panthers-thank.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/core-app-api': major
```

Comment:
Not a major change - leave this as `patch`

Reply:
Done